scripts/ndk-make.sh

```diff
@@ -108,12 +108,14 @@ if test -z "$(find "$ANDROID_NDK_ROOT" -name libgcc.a -print -quit)"; then
     echo 'INPUT(-lunwind)' >"$TMPLIB/libgcc.a"
 fi
 
+RUSTUP_TOOLCHAIN=1.64.0
```

Comment:
maybe add a comment that when modifying this line, one should also modify `install-toolchains.sh`. and maybe also say why we are not using `cat rust-toolchain` - at least i tend to forget about that completely as it is touched only very rarely - and at least some other devs are not that deep in rust :)